Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_riff_floor_is_lava.nut

```diff
@@ -36,7 +36,18 @@ void function CreateCustomSpawns_Threaded()
 {
 	WaitEndFrame() // wait for spawns to clear
 	
-	float raycastTop = GetLethalFogTop() + 2500.0
+	string thismap = GetMapName()
+	
+	float raycastTop = GetLethalFogTop() // OG 2500, Lf 545, TDM 3500
+	if ( ( thismap.find( "_lf_" ) ) || ( thismap.find( "_coliseum" ) ) ) // coliseum is only there because I can. Maps like boomtown (grave) already let the ray cast through the ceiling
+		raycastTop = raycastTop + 545 // 545
+	else if ( thismap.find( "_grave" ) )
+		raycastTop = raycastTop + 1900
+	else
+		raycastTop = raycastTop + 2650
+
+		print( "\n\n\n\n\n\n				RAYCASTTOP:" + raycastTop + "\n\n\n\n\n\n" ) // this has to be higher than what you want to spawn on. May spawn in wall if lower than some rooftops
```

Comment:
```suggestion
```

Reply:
Sorry I did not respond to this, my first time with a Pr. I placed that print for a future where custom maps may require people to double-check the value used. It just prints once per map load and just makes some space in the log to make it easy to see. That future may never exist though, thanks.